src/main/java/com/terraformersmc/cinderscapes/block/PyracinthBlock.java

```diff
@@ -29,4 +30,11 @@ public void randomDisplayTick(BlockState state, World world, BlockPos pos, Rando
             world.addParticle(ParticleTypes.FLAME, pos.getX() + 0.4f + random.nextFloat() * 0.2f, pos.getY() + 0.5f, pos.getZ() + 0.4f + random.nextFloat() * 0.2f, 0.0f, 0.05f, 0.0f);
```

Comment:
Let's also apply the model offset to the spawn particles method here, so the center of the particles appears as the center of the block model.